pkg/bisect/minimize/slice.go

```diff
@@ -47,13 +47,14 @@ func Slice[T any](config Config[T], slice []T) ([]T, error) {
 
 type sliceCtx[T any] struct {
 	Config[T]
-	chunks   []*arrayChunk[T]
-	predRuns int
+	chunks   []*arrayChunk[T] // nolint:all to ignore unused
```

Comment:
Does it work if you add it on top of `type sliceCtx`?
Marking each field in templates does not look scalable.
One annotation on top of the struct is less ugly.
Also, is it possible to ignore just the error produced rather than "all"? I assume there is only 1 buggy linter, or there are more?

Reply:
I'm still trying to find the guilty linter. Please consider it a WIP.
Thanks for "move to struct level".